src/assets/styles/components/multi-select.scss

```diff
@@ -19,4 +19,10 @@
       box-shadow: 0 0 0 0.2rem rgba(0,123,255,.25)
     }
   }
+
+  .ng-optgroup {
+    font-weight: 700 !important;
```

Comment:
do we need importants here?

Reply:
Unfortunately on a couple of these we do as the ng-select module's theme will overwrite the custom styles without it. I tried targeting these with custom classes, but due to how nested some of these elements are I was unable to reach them with a custom class.